Cairo Desktop/Cairo Desktop/MenuBar.xaml.cs

```diff
@@ -862,8 +862,29 @@ private void SysSleep(object sender, RoutedEventArgs e)
 
         private void InitCairoSettingsWindow(object sender, RoutedEventArgs e)
         {
-            CairoSettingsWindow window = new CairoSettingsWindow();
```

Comment:
Change this section to:
```
CairoSettingsWindow.Instance.Show();
CairoSettingsWindow.Instance.Activate();
```


Add to CairoSettingsWindow:
```
private static CairoSettingsWindow _instance = null;
public static CairoSettingsWindow Instance
{
   get
   {
      if (_instance == null)
      {
         _instance = new CairoSettingsWindow();
      }
     
      return _instance;
      }
}

```

Add static constructor and change the public constructor to private:
```
static CairoSettingsWindow() { }
private CairoSettingsWindow()
```

Reply:
![Screenshot (20200114-015018)](https://user-images.githubusercontent.com/5261273/72304325-81caa580-3670-11ea-97cc-ee1458e365e8.png)
Your solution, raises the same error that I get in my solution.

Reply:
It seams like that instance field must be returning null. There may be a race condition. Between a null check and when a previous settings window has closed. I will run some tests in my repo

Reply:
Field Istance is not null...
But report explain that is not possible call .Show after closing.

Reply:
Ahhhhh. Make change to CairoSettingsWindow Closing 

  ```
      private void Window_Closing(object sender, System.ComponentModel.CancelEventArgs e)
        {
            saveChanges();

            **_instance = null;**
        }
```

Reply:
You!
It is perfect now.
I do some test and push.

Ty for you big help!  ;-)

Hem... Why it must be null?

Reply:
The window starts disposing of its resources when the Closing call doesn't get canceled.
If we null it out then the next call to instance will create a new window. The previous one will get cleaned up by the Garbage Collector